styx-cli-unshaded/src/test/java/com/spotify/styx/cli/CliMainTest.java

```diff
@@ -494,6 +511,8 @@ public void testBackfillList() {
         .description("Description")
         .nextTrigger(Instant.parse("2017-01-01T00:00:00Z"))
         .schedule(Schedule.DAYS)
+        .created(currentTime)
+        .lastModified(currentTime)
```

Comment:
Let's add a test also where we print out an "old" backfill that doesn't have those timestamps set.

Reply:
New test case testBackfillListWithoutCreatedTS added